Update release-note with GraphQL Changes (#2175)
ballerina-platform/ballerina-release: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md

#### `graphql` package

- Added support for disabling introspection queries
- Added support for GraphQL interfaces
- Added support for interfaces implementing interfaces
- Introduced GraphQL client configurations

### Improvements

#### `graphql` package

> - Added service-level interceptor execution for records fields, maps & tables

Reviewer: ```suggestion
- Added service-level interceptor execution for records fields, maps, and tables
```

- Added service-level interceptor execution for subscriptions

> - Added returning all the errors related to a GraphQL document in a single response

Reviewer: ```suggestion
- Enabled returning all the errors related to a GraphQL document in a single response
```

Reviewer 2: `Enabled` does not convey the expression here. This is a change to the existing behavior, which returned responses separately for different validation steps. `Enabled` sounds like this is something we have `disabled` previously. Which isn't the case.

Reviewer 2: ```suggestion
- Changed to return all the errors related to a GraphQL document in a single response
```

Might be a better option.

Reviewer: Shall we say  `Added the ability to return all the errors related to a GraphQL document in a single response`?

Reviewer 2: Still not perfect. This is the behavior now. There's no option to the user to change this behavior.

Reviewer: +1 for this: `Changed to return all the errors related to a GraphQL document in a single response`
